src/gui/viewer.cpp

```diff
@@ -499,6 +574,13 @@ void InfoWidget::update_info(VTFLib::CVTFFile* file) {
 			break;
 		}
 	}
+
+	for (int i = 0; i < 7; i++) {
+		if (i == file->GetMinorVersion()) {
+			versionCombo_->setCurrentIndex(i);
+			break;
+		}
+	}
```

Comment:
```suggestion
	versionCombo_->setCurrentIndex(file->GetMinorVersion());
```

Add a clamp if you want, but I think there's a version check before this